packages/app-store/routing-forms/components/react-awesome-query-builder/config/BasicConfig.ts

```diff
@@ -356,8 +356,8 @@ const types: Types = {
       },
       multiselect: {
         operators: [
-          //   "select_any_in",
-          //   "select_not_any_in",
+          "select_any_in",
+          "select_not_any_in",
```

Comment:
I accidentally commented these thinking that I was removing them for multiselect field but it is actually a case of `Select` field which is suppsoed to have `any_in` and `not_any_in` operators that use multiselect widget for operand.